### PR TITLE
sync 0.17.x branch

### PR DESCRIPTION
**Pull request by LucaGuerra**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area driver-kmod

/area driver-bpf

/area driver-modern-bpf

/area libsinsp

**Does this PR require a change in the driver versions?**

/version driver-API-version-patch

**What this PR does / why we need it**:

Sync 0.17.x release branch with the following changes:
* https://github.com/falcosecurity/libs/pull/1910
* https://github.com/falcosecurity/libs/pull/1942
* https://github.com/falcosecurity/libs/pull/1953
* https://github.com/falcosecurity/libs/pull/1961
* https://github.com/falcosecurity/libs/pull/1960
* https://github.com/falcosecurity/libs/pull/1899
* https://github.com/falcosecurity/libs/pull/1940
* https://github.com/falcosecurity/libs/pull/1946
* https://github.com/falcosecurity/libs/pull/1979
* https://github.com/falcosecurity/libs/pull/1895
* https://github.com/falcosecurity/libs/pull/1922
* https://github.com/falcosecurity/libs/pull/1923
* https://github.com/falcosecurity/libs/pull/1959

~Also, increment driver API version because we are patching it (@FedeDP please double check that this is correct).~ API version does not need to be incremented

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by FedeDP** [comment]
I think we also need https://github.com/falcosecurity/libs/pull/1940 for windows related CI jobs.

**Comment by FedeDP** [comment]
Added also #1980 to the 0.17.3 milestone.

**Comment by FedeDP** [comment]
Moved also https://github.com/falcosecurity/libs/pull/1946 to 0.17.3 milestone to fix 
>  Drivers CI Build / build-scap-open-w-extern-bpf-skeleton (pull_request) Failing after 10s

File: driver/API_VERSION
Comment (FedeDP) [inline driver/API_VERSION:1]: I don't think we need this unless we want an API_VERSION on the release branch that is greater than on master.

**Comment by LucaGuerra** [comment]
Thanks @FedeDP , added everything

**Review by FedeDP — APPROVED** [review]
/approve

**Review by Andreagit97 — APPROVED** [review]
/approve

**Pull request merged**